### PR TITLE
add org.dash.electrum.electrum_dash app

### PR DESCRIPTION
**Pull request by zebra-lucky**
Please look new app PR: `org.dash.electrum.electrum_dash`
This is a fork of BTC version `org.electrum.electrum`

### CODE REVIEW

File: org.dash.electrum.electrum_dash.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    "tag": "3.3.8.5",
+                    "commit": "b52b5edef6263226ef9dd7cdcea815147408cde5"
+                },
+                {
+                    "type": "patch",
+                    "path": "electrum-dash-setup.patch"
+                },
+                {
+                    "type": "file",
+                    "path": "org.dash.electrum.electrum_dash.appdata.xml"
```
Comment (LeandroStanger) [inline org.dash.electrum.electrum_dash.json:58]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by LeandroStanger** [comment]
bot, build org.dash.electrum.electrum_dash

File: org.dash.electrum.electrum_dash.json (hunk 1 above)
Comment (zebra-lucky) [reply org.dash.electrum.electrum_dash.json:58]: I read what needs to be fixed.

Comment (zebra-lucky) [reply org.dash.electrum.electrum_dash.json]: added some fixes with 0763cde:
- upgrade appdata to metainfo
- change screenshots
- add translate url
- build from sdist archive to include translations

**Comment by LeandroStanger** [comment]
bot, build org.dash.electrum.electrum_dash

**Comment by zebra-lucky** [comment]
Seems some tech problem with build:
> fatal: unable to access 'https://github.com/flathub/flathub/': The requested URL returned error: 500

**Comment by barthalion** [comment]
bot, build org.dash.electrum.electrum_dash

File: org.dash.electrum.electrum_dash.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline org.dash.electrum.electrum_dash.metainfo.xml:1]: It should be upstreamed.

**Comment by barthalion** [comment]
Have you contacted upstream developer about submitting their app to Flathub? Ideally they should co-maintain it.

**Comment by zebra-lucky** [comment]
Yes, there is issue related to this PR:
https://github.com/akhavr/electrum-dash/issues/92

File: org.dash.electrum.electrum_dash.metainfo.xml (hunk 2 above)
Comment (zebra-lucky) [reply org.dash.electrum.electrum_dash.metainfo.xml:1]: Not sure what I should change, reading docs.

Comment (zebra-lucky) [reply org.dash.electrum.electrum_dash.metainfo.xml:1]: Do you mean appdata should be added to upstream?

Comment (barthalion) [reply org.dash.electrum.electrum_dash.metainfo.xml:1]: Yes, that would be perfect.

**Comment by barthalion** [comment]
/merge

Let us know if someone else from the team should be added to the new repo. Thanks!

File: org.dash.electrum.electrum_dash.metainfo.xml (hunk 2 above)
Comment (zebra-lucky) [reply org.dash.electrum.electrum_dash.metainfo.xml:1]: I'll ask to include AppData in the next relase, in https://github.com/akhavr/electrum-dash/issues/92

**Comment by zebra-lucky** [comment]
Is merge fails for some reason?

**Comment by barthalion** [comment]
Looks like it. Let's try again.

/merge

**Comment by zebra-lucky** [comment]
Thank you!

**Comment by zebra-lucky** [comment]
> Let us know if someone else from the team should be added to the new repo. Thanks!

Please add https://github.com/akhavr/ to https://github.com/flathub/org.dash.electrum.electrum_dash

**Comment by bilelmoussaoui** [comment]
> > Let us know if someone else from the team should be added to the new repo. Thanks!
> 
> Please add https://github.com/akhavr/ to https://github.com/flathub/org.dash.electrum.electrum_dash

Done

**Comment by zebra-lucky** [comment]
> > > Let us know if someone else from the team should be added to the new repo. Thanks!
> > 
> > 
> > Please add https://github.com/akhavr/ to https://github.com/flathub/org.dash.electrum.electrum_dash
> 
> Done

Thank you!

**Pull request closed without merging**